postalicious/app.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express')
+const app = express()
+
+app.use(express.static(__dirname))
+
+app.get('/', (request, response) => {
+  response.render('./index.html')
```

Comment:
I think you need to use `sendFile` here. I'm not sure rendor is intended to be used this way. You sure this works?